tsb/mp/main.tf

```diff
@@ -55,34 +39,9 @@ module "tsb_mp" {
   es_username                = module.es.es_username
   es_password                = module.es.es_password
   es_cacert                  = module.es.es_cacert
-  registry                   = local.infra[var.tsb_mp["cloud"]][var.tsb_mp["cluster_id"]]["outputs"].registry
-  cluster_name               = local.infra[var.tsb_mp["cloud"]][var.tsb_mp["cluster_id"]]["outputs"].cluster_name
-  k8s_host                   = local.infra[var.tsb_mp["cloud"]][var.tsb_mp["cluster_id"]]["outputs"].host
-  k8s_cluster_ca_certificate = local.infra[var.tsb_mp["cloud"]][var.tsb_mp["cluster_id"]]["outputs"].cluster_ca_certificate
-  k8s_client_token           = local.infra[var.tsb_mp["cloud"]][var.tsb_mp["cluster_id"]]["outputs"].token
-
-}
-
-module "aws_register_fqdn" {
-  count    = var.dns_provider == "aws" ? 1 : 0
-  source   = "../../modules/aws/register_fqdn"
-  dns_zone = "cx.tetrate.info"
-  fqdn     = var.tsb_fqdn
-  address  = module.tsb_mp.ingress_ip != "" ? module.tsb_mp.ingress_ip : module.tsb_mp.ingress_hostname
-}
-
-module "azure_register_fqdn" {
-  count    = var.dns_provider == "azure" ? 1 : 0
-  source   = "../../modules/azure/register_fqdn"
-  dns_zone = "azure.cx.tetrate.info"
-  fqdn     = var.tsb_fqdn
-  address  = module.tsb_mp.ingress_ip != "" ? module.tsb_mp.ingress_ip : module.tsb_mp.ingress_hostname
-}
-
-module "gcp_register_fqdn" {
```

Comment:
The register modules are now called directly from the Makefile to be able to only require the ones that match the cloud being used and do not require any additional local config for other clouds.